taxi/models.py

```diff
@@ -0,0 +1,30 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+    def __str__(self):
+        return self.name
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=255, unique=True)
+
+    class Meta:
+        ordering = ('license_number',)
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=255)
+    manufactured = models.ForeignKey(
```

Comment:
manufactureR field name is better here
maybe you meant "mafunactured by", then it should be "manufactired_by"